quint/src/generated/QuintListener.ts

```diff
@@ -1,6 +1,10 @@
 // Generated from ./src/generated/Quint.g4 by ANTLR 4.9.0-SNAPSHOT
 
 
+    // Used for forming errors
+    import { quintErrorToString } from '../quintError'
```

Comment:
Unusual indent here. Should we dedent it in `Quint.g4`? I'm okay either way.

Reply:
Oh yeah, I didn't look at the generated code! I think removing the indent won't hurt readability of the antlr, so I'll do that.